src/core/server/bootstrap.ts

```diff
@@ -82,7 +82,7 @@ export async function bootstrap({
   const env = Env.createDefault(REPO_ROOT, {
     configs,
     cliArgs,
-    isDevClusterMaster: isMaster && cliArgs.dev && features.isClusterModeSupported,
```

Comment:
is the update to `isMaster` covered by a different issue/PR?

Reply:
Updated! I've added alias to the 'isMaster' referenced from @types/node modules.

Reply:
Do we need to change or use alias for `isMaster`? I think it is defined in node cluster index which is not the code we write/control. If it is something we import, do we have to rename?

Reference: https://nodejs.org/docs/latest-v14.x/api/cluster.html#cluster_cluster_ismaster